frappe/core/doctype/data_import/data_import.json

```diff
@@ -167,11 +169,25 @@
    "hidden": 1,
    "label": "Payload Count",
    "read_only": 1
+  },
+  {
+   "default": ",;\\t|",
+   "depends_on": "custom_delimiters",
+   "description": "If your CSV uses a different delimiter, add that character here, ensuring no spaces or additional characters are included.",
+   "fieldname": "delimiter_options",
+   "fieldtype": "Data",
+   "label": "Delimiter options"
+  },
+  {
+   "default": "0",
+   "fieldname": "custom_delimiters",
+   "fieldtype": "Check",
+   "label": "Custom delimiters"
```

Comment:
Can just drop this checkbox since its just visual? All the delimiters that are specified as the defaults above will be used even if this isn't checked

Reply:
From UX perspective: originally I just put the delimiter options field, but it looked ugly.

On a more serious note:
This checkbox is designed to hide the delimiter options data field when unchecked by default.
My reasoning behind this is, that we avoid confusing regular users who might be tempted to modify or delete the default parameters (,;\t|) without fully understanding the consequences. Modifying these delimiters without proper knowledge could lead to unexpected behavior and potential errors.

I'd expect that ~99% of the users won't need to adjust the delimiter options. For the remaining ~1%  of advanced users who understand the purpose of the delimiter options and need to modify them, they can simply check the checkbox to reveal the field. Or the other case, if a regular user encounters an error message related to delimiters, the message will explicitly instruct them to update the delimiter options, guiding them to the appropriate setting.